app/views/pqs/_com_data.html.slim

```diff
@@ -41,7 +41,7 @@ hr/
           = link_to raw("<i class=\"fa fa-envelope-o\"></i> Send reminder#{reminder_accept_count}"), {controller: 'action_officer_reminder', action: 'accept_reject', id: ao_pq.id, remote: true}, {class: 'ao-reminder-link', title: 'Send an accept/reject reminder email'}
         = link_to 'Manually reject this action officer', {controller: 'manual_reject_commission', action: 'reject_manual', id: ao_pq.id}, {class: 'button-secondary', title: 'Reject the question manually'}
     - unless @pq.is_new?
-      = render partial: 'shared/action_officer_selection', locals: {question: @pq, form: f, reassign: true}
+      = render partial: 'shared/action_officer_selection', locals: {action_officers: @action_officers, question: @pq, form: f, reassign: true}
```

Comment:
isn't there a local action_officers variable passed in to _com_data, which should be passed on to the next partial, rather than the instance variable @action_officers?